src/tactic/monotonicity/lemmas.lean

```diff
@@ -69,4 +70,10 @@ end
 
 open set
 
-attribute [mono] monotone_inter monotone_union bUnion_mono sUnion_mono seq_mono monotone_prod
+attribute [mono] monotone_inter monotone_union
+                 sUnion_mono bUnion_mono sInter_subset_sInter bInter_mono
```

Comment:
```suggestion
                 sUnion_mono bUnion_mono sInter_subset_sInter bInter_mono'
```

Reply:
The mono tactic is not smart enough to use that, sorry.

Reply:
I didn't know that it is not smart enough.